src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -715,7 +716,11 @@ private function getTraceKey(Request $request): string
             $path .= '?'.$qs;
         }
 
-        return $request->getMethod().' '.$path;
+        try {
+            return $request->getMethod().' '.$path;
+        } catch (SuspiciousOperationException $e) {
+            return '_INVALID_ '.$path;
```

Comment:
to make it clear it's the method?
```suggestion
            return '_BAD_METHOD_ '.$path;
```